File: app/src/main/java/restaurant/sa/com/sarestaurant/appview/restaurant/model/Viewport.java

package restaurant.sa.com.sarestaurant.appview.restaurant.model;

import com.google.gson.annotations.Expose;

public class Viewport {

    @Expose
    private Northeast northeast;
    @Expose
    private Southwest southwest;

    public Northeast getNortheast() {
        return northeast;
    }

    public void setNortheast(Northeast northeast) {
        this.northeast = northeast;
    }

    public Southwest getSouthwest() {
        return southwest;
    }

    public void setSouthwest(Southwest southwest) {
        this.southwest = southwest;
    }

}

File: app/src/main/java/restaurant/sa/com/sarestaurant/appview/restaurant/model/Viewport.kt
package restaurant.sa.com.sarestaurant.appview.restaurant.model

import android.arch.persistence.room.ColumnInfo
import android.arch.persistence.room.Embedded
import android.arch.persistence.room.Entity
import android.arch.persistence.room.PrimaryKey
import com.google.gson.annotations.Expose

@Entity(tableName = "Viewport")
data class Viewport(
        @Embedded var northeast: Northeast?,
        @Embedded var southwest: Southwest?)

